transifex/openformats
# Update CircleCI config to v2

Problem and/or solution
-----------------------
We need to migrate to CircleCI 2, as v1 is deprecated.

Note: the step that uploaded the documentation on readthedocs.org was removed. It didn't work for some time anyway, since this endpoint was deprecated: http://readthedocs.org/build/openformats.

We can revisit the deployment part on a future PR.

How to test
-----------
Check if the tests run properly for this PR.

Reviewer checklist
------------------

Code:

PR:
